wildfly-standalone-centos8/README.md

```diff
@@ -109,7 +109,7 @@ The deployment takes less than 10 minutes to complete.
 
 ## Troubleshoot
 
-This section is to give you an idea on troubleshooting the issues that you might face when you deploy this template. If the parameter criteria are not fulfilled(like the Admin Password criteria) or if any parameter that are mandatory are not filled in the parameters section then the deployment will not start. Also make sure that you accept the terms and conditions mentioned before you click on Purchase. Once the deployment starts you can clearly see the resources getting deployed on the deployment page and if any deployment fails you can see which of the resources failed and check the deployment failure message for more details. If your deployment fails due to the script extension, you can see that the VM is already deployed so please login to the VM to check the logs for more troubleshooting. The logs are stored in the file wildfly.install.log in path /var/lib/waagent/custom-script/download/0. The script can fail due to various reasons like incorrect RHSM Username/Password. This log file gives you a clear message on why the script failed and exited.  Use these logs to detect and correct errors.  Run the following commands to check the logs once you login to the VM.
+This section provides guidance on issues you may run into when deploying this template. If any of the parameter criteria is not provided (ex. -  the Admin Password criteria) or if any mandatory parameter is not provided in the parameters section, the deployment will not start. Also ensure the Terms and Conditions are accepted before clicking Purchase.  Once the deployment starts you will see the resources being deployed on the deployment page.  If any deployment fails you will see which of the resources failed and you can check the deployment failure message for more details. If your deployment fails due to the script extension, you will see that the VM is already deployed so login to the VM to check the logs for additional troubleshooting. The logs are stored in the file wildfly.install.log in path /var/lib/waagent/custom-script/download/0. The script can fail due to various reasons like incorrect RHSM Username/Password. This log file gives you a clear message on why the script failed and exited.  Use these logs to detect and correct errors.  Run the following commands to check the logs once you login to the VM.
```

Comment:
Change accommodated - Under the “Troubleshooting” section. Made a few other minor changes. Please review and provide us feedback.